src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -450,6 +452,27 @@ def range_statement(start: int, end: int, step: int, column_name: str) -> str:
     )
 
 
+def schema_query_for_values_statement(output: List[Attribute]) -> str:
+    data_types = [attr.datatype for attr in output]
+    names = [quote_name(attr.name) for attr in output]
+    nullables = [attr.nullable for attr in output]
+    cells = [
+        schema_expression(data_type, is_nullable)
+        for data_type, is_nullable in zip(data_types, nullables)
+    ]
```

Comment:
Extracting the info into separate lists and zip them here feels a bit counterintuitive. Should we just do a `for attr in output`?